src/lib.rs

```diff
@@ -929,6 +927,20 @@ impl BoxConstraints {
     pub fn is_height_bounded(&self) -> bool {
         self.max.height.is_finite()
     }
+
+    /// Check to see if these constraints are legit.
+    pub fn check(&self, name: &str) {
+        if !(0.0 <= self.min.width
+            && self.min.width <= self.max.width
+            && self.max.width <= std::f64::INFINITY)
```

Comment:
Checking less than or equal to infinity seems strange to me; the only thing that wouldn't is a NaN, I believe.

Reply:
This is pretty funny now that I think about it.